invenio_oauthclient/views/client.py

```diff
@@ -60,8 +60,13 @@ def auto_redirect_login(*args, **kwargs):
     )
     would_redirect = auto_redirect_enabled and not local_login_enabled
     remote_apps = list(current_oauthclient.oauth.remote_apps)
+    remote_app_configs = current_app.config["OAUTHCLIENT_REMOTE_APPS"]
 
-    if would_redirect and len(remote_apps) == 1:
+    if (
+        would_redirect
+        and len(remote_apps) == 1
+        and not remote_app_configs[0].get("hide", False)
+    ):
```

Comment:
AFAIK, the `hide` flag wasn't really exposed before this PR so i doubt that this makes a difference to anybody (i expect all configured `REMOTE_APPS` to be visible).

would it make sense here to filter the `remote_apps` and `remote_app_configs` to only the visible ones here before this check?
otherwise, if you have one hidden and one visible remote app configured, this logic would never trigger (even though i would expect it to?)

Reply:
Thank you, good catch. Adapted it to perform the check on visible remote apps only